itest/rust/src/variant_test.rs

```diff
@@ -334,6 +334,24 @@ fn variant_type_correct() {
     );
 }
 
+#[itest(focus)]
+fn variant_stringify_correct() {
+    assert_eq!(Variant::from("value").stringify(), gstr("value"));
+    assert_eq!(Variant::nil().stringify(), gstr("<null>"));
+    assert_eq!(true.to_variant().stringify(), gstr("true"));
+    assert_eq!(30.to_variant().stringify(), gstr("30"));
```

Comment:
You can use `Variant::from()` here, too.

Reply:
is this informational or is it better to use Variant::from ?